### ISSUE TITLE
Data hacking to create overall network visualization of researcher relationships OR citation network

### ISSUE THREAD

**Issue by jasonclark** [jasonclark/linked-people-bioscience#8]
There are about 35 researchers that form the network included in this release. Any ideas for these specific visualizations are welcome. **Brainstorming and sketching without code are welcome.**

Each researcher has a profile page which has some nice data (rdf, json-ld) behind the scenes in the markup. 

This includes structured data for the page.
[https://arc.lib.montana.edu/linked-bioscience/about.json?id=2](https://arc.lib.montana.edu/linked-bioscience/about.json?id=2)

And structured data for the person.
[https://arc.lib.montana.edu/linked-bioscience/person.json?id=2](https://arc.lib.montana.edu/linked-bioscience/person.json?id=2)

And an indexable/spiderable list of all json-ld files is available in a sitemap.
[https://arc.lib.montana.edu/linked-bioscience/sitemap-list-jsonld.xml](https://arc.lib.montana.edu/linked-bioscience/sitemap-list-jsonld.xml)

Those are open data endpoints that can be used to power a research network visualization.

I also have a list of publications produced by these researchers since 2008. It is in a .CSV format and can be a data source for a visualization related to the works created by the group. 

[Publication list as .csv](http://shelf.lib.montana.edu/linked-bioscience/about.json?id=2)

**Comment by kimpham54** [comment]
@jasonclark tried to look at the shelf.lib.montana.edu links, they require sign in!

**Comment by jasonclark** [comment]
Sorry @kimpham54! Those links are also live and now fixed. I'll be online all day tomorrow if you have other questions.

**Comment by annakrystalli** [comment]
Hi @jasonclark! I'll be honest and say my main skills are in R so I was just wondering if you were aware of R packages making interactive network visualization really simple? ie if you get the json data into R (which should be easily doable) it's really simple to then build the network with a few lines of code. I've worked with such networks in R so would be happy to help if you decided it's of interest. Have a look especially at [visNetwork](http://www.htmlwidgets.org/showcase_visNetwork.html) and [networkD3](http://www.htmlwidgets.org/showcase_networkD3.html) widgets.

**Comment by jasonclark** [comment]
Hi, @annakrystalli! Thanks for reaching out. I think I was aware that other languages had some visualization packages, but these R examples are really great. I think your idea for a demo network visualization using R could be really interesting to show how adaptable the JSON is for other packages. I think it could really help illustrate the "open data/open science" component of the project. Maybe we could add a /samples/ directory and have the R scripts in there. I have never used R, so would love your help there. Should I start at looking at the JSON format that R requires? Feel free to submit a pull request and add a directory where we might add these external examples to other software packages. And if you have an example script you could drop it in there. Let me know if this makes sense. Talk more soon.

**Comment by jasonclark** [comment]
@annakrystalli I'm also going to look at how R can do the heavy-lifting for the visualization and then generate the visualization as HTML. Will share what I learn. If you already know how to do this, let me know. More as things develop...

**Comment by annakrystalli** [comment]
@jasonclark yes I do know how to do this. Unfortunately (annoyingly) the networks I've been working on are not public yet but! I will be creating a dummy version of them within the next couple of weeks to present the visualisation aspect so will share that with you. In the mean time, the first thing to do would be to get the data in the right shape, either an n x n matrix (n the number of researchers) with 1 indicating a publication link between researchers and 0 for no link or a two column data.frame where each row contains the identity of two nodes (researchers) which share an edge (link). Does this make sense?
